mantidimaging/gui/widgets/spectrum_widgets/fitting_display_widget.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2021 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+from collections.abc import Sequence
+import numpy as np
+from PyQt5.QtWidgets import QWidget, QVBoxLayout
+from mantidimaging.gui.windows.spectrum_viewer.spectrum_widget import SpectrumPlotWidget
+
+
+class FittingDisplayWidget(QWidget):
+    """
+    Widget for displaying fitting-related spectrum plot using the reusable SpectrumPlotWidget.
+    """
+
+    def __init__(self, parent: QWidget | None = None) -> None:
+        super().__init__(parent)
+
+        self.layout: QVBoxLayout = QVBoxLayout(self)
+        self.spectrum_plot: SpectrumPlotWidget = SpectrumPlotWidget()
+        self.layout.addWidget(self.spectrum_plot)
+
+    def update_plot(self,
+                    x_data: Sequence[float] | np.ndarray,
+                    y_data: Sequence[float] | np.ndarray,
```

Comment:
Can this be `def update_plot(self, x_data: np.ndarray, y_data: np.ndarray, label: str = "ROI") -> None:`
I looks like its always called with the output of `self.model.get_spectrum()` which is an ndarray.